code/modules/jobs/job_types/coroner.dm

```diff
@@ -0,0 +1,63 @@
+/datum/job/coroner
+	title = JOB_CORONER
+	description = "Perform Autopsies whenever needed, \
+		Update medical records accordingly, apply formaldehyde."
+	department_head = list(JOB_CHIEF_MEDICAL_OFFICER)
+	faction = FACTION_STATION
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the Head of Personnel and the Chief Medical Officer"
+	exp_granted_type = EXP_TYPE_CREW
+	config_tag = "CORONER"
+
+	outfit = /datum/outfit/job/coroner
+	plasmaman_outfit = /datum/outfit/plasmaman/medical
+
+	paycheck = PAYCHECK_CREW
+	paycheck_department = ACCOUNT_MED
+
+	display_order = JOB_DISPLAY_ORDER_CORONER
+	bounty_types = CIV_JOB_MED
+	department_for_prefs = /datum/job_department/medical
+	departments_list = list(
+		/datum/job_department/medical,
+		/datum/job_department/service,
+		)
+
+	mail_goodies = list(
+		/obj/item/reagent_containers/cup/bottle/formaldehyde = 30,
+		/obj/item/storage/box/bodybags = 15,
+		/obj/item/healthanalyzer = 10,
+		/obj/effect/spawner/random/medical/organs = 5,
+		/obj/effect/spawner/random/medical/memeorgans = 1,
+	)
+
+	family_heirlooms = list(/obj/item/clothing/head/helmet/skull, /obj/item/table_clock)
+
+	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
+
+	rpg_title = "Plague Doctor"
```

Comment:
I didn't, I didnt know Viro had that name. I chose it because from what I know, plague doctors were fuckall useless, bled you out, and told you you're done for

Reply:
that is SO cap junior